promptsource/templates/anli/templates.yaml

```diff
@@ -1,27 +1,38 @@
 dataset: anli
 templates:
-  391acb43-732a-4c09-8c10-e02d226d8854: !Template
+  0cc3ae39-3997-4686-8c93-5d51457efa1f: !Template
     answer_choices:
-    - 'No'
-    - Neutral
-    - 'Yes'
+    - correct
+    - inconclusive
+    - incorrect
     answer_choices_key: null
-    id: 391acb43-732a-4c09-8c10-e02d226d8854
-    jinja: 'Sentence 1: {{premise}}
-
-      Sentence 2: {{hypothesis}}
-
-      Question: Does Sentence 1 contradict Sentence 2? Yes, No, or Neutral? |||
-
-      {{answer_choices[label]}}'
+    id: 0cc3ae39-3997-4686-8c93-5d51457efa1f
+    jinja: '{{premise}} Using only the above description and what you know about the
```

Comment:
Should there be newlines after premise here and elsewhere? Not sure - wondering if we have any indication of whether it matters.

Reply:
Sadly we just didn't have enough templates in D3 to tell whether this matters. I threw in a mix of with and without newlines this time so hopefully we will figure out soon.